var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -46,5 +46,9 @@ def cmake_args(self):
             options.extend([
                 '-DENABLE_CUDA=On',
                 '-DCUDA_TOOLKIT_ROOT_DIR=%s' % (spec['cuda'].prefix)])
+            if 'cuda_arch' in spec.variants:
```

Comment:
This condition will always evaluate to true, since the variant is present for all `CudaPackage`s.

Reply:
Does this default  to something if not  provided?